docs/terminal/melbourne.md

```diff
@@ -121,40 +120,43 @@ Due to the low level of CTA above Avalon, aircraft conducting approaches will co
 Departing aircraft generally require an airways clearance on the ground, due to the low level of CTA.  Aircraft should report taxiing to the TCU controller, who will issue a squawk code and traffic statement.  These aircraft should be instructed to report at the holding point, where airways clearance will be issued.
 
 !!! example
-    **AM318**: "Melbourne Approach, AM318, King Air IFR, taxiing YMAV runway 18 for YMEN, 2 POB" `AIP GEN 3.4`  
+    **AM318**: "Melbourne Approach, AM318, King Air, POB 2, IFR, taxiing YMAV runway 18 for YMEN"
     **ML TCU**: "AM318, Melbourne Approach, squawk 3542, no reported IFR traffic, report ready at the holding point for airways clearance"  
     **AM318**: "Squawk 3542, wilco, AM318"  
     
     **AM318**: "AM318, ready runway 18"  
     **ML TCU**: "AM318, cleared to YMEN direct, AV5 departure, climb via SID to A040"  
     **AM318**: "Cleared to YMEN direct, AV5 departure, climb via SID to A040, AM318"  
     **ML TCU**: "AM318, assigned heading left 020, report airborne"  
-    **AM318**: "Assigned left 020, wilco, AM318"  
+    **AM318**: "Assigned left 020, wilco, AM318"    
+    `AIP GEN 3.4`
 
-## YMMB TWR Offline
```

Comment:
YMMB Offline is here, but not YMMB online

Reply:
MB TWR coord section?